### PR TITLE
Cryo loadout changes + locker stuff

### PR DESCRIPTION
**Pull request by DexterDude**
Makes it so the Platoon Commander does not start equipped anymore. I assume this was the case because rounds were much quicker in the olden days and they had to go to their office to get their fax quick. This change promotes platoon commanders eating with their marines.
Removes satchels on spawn for all cryo presets. Now that loadout items spawn in lockers, there's no need for satchels.
Removes un-needed weapons from platoon sergeant lockers. Considering they gear up with their marines, they don't need it. UPP platsarge guns stay because they don't gear up with everyone else.
Kills squad sergeant snowflake helmets. They kinda look ugly and aren't needed, I've kept the platoon sergeant helmet so he remains distinct.

### CODE REVIEW

File: code/modules/gear_presets/uscm_ship.dm [hunk 1]
```diff
@@ -575,36 +575,46 @@
 	minimap_background = MINIMAP_ICON_BACKGROUND_CIC
 	var/access_list = ACCESS_LIST_MARINE_MAIN
 
+/datum/equipment_preset/uscm_ship/so/load_status(mob/living/carbon/human/new_human)
```
Comment (AndroBetel) [inline code/modules/gear_presets/uscm_ship.dm:578]: these two proc definitions can be easily put as one

File: code/modules/gear_presets/uscm_ship.dm [hunk 2]
```diff
@@ -613,20 +623,30 @@
 	faction = FACTION_UPP
 	access_list = ACCESS_LIST_UPP_PLATOON
 
-/datum/equipment_preset/uscm_ship/so/upp/load_gear(mob/living/carbon/human/new_human)
+/datum/equipment_preset/uscm_ship/so/upp/load_status(mob/living/carbon/human/new_human)
```
Comment (AndroBetel) [inline code/modules/gear_presets/uscm_ship.dm:626]: unneeded, inherits the behavior from parent class

**Review by AndroBetel — CHANGES_REQUESTED** [review]

**Review by AndroBetel — APPROVED** [review]

File: code/modules/gear_presets/uscm_ship.dm [hunk 3]
```diff
@@ -579,32 +579,39 @@
...
+	new_human.nutrition = NUTRITION_VERYLOW
+	if(!new_human.client)
+		return
+
+	add_verb(new_human.client, /client/proc/commander_rename_platoon)
+
+/datum/equipment_preset/uscm_ship/so/lesser_rank
+	paygrade = "MO1"
+
+/datum/equipment_preset/uscm_ship/so/equipped
```
Comment (AndroBetel) [inline code/modules/gear_presets/uscm_ship.dm:593]: lt is now naked in lobby... fail. 
image

File: code/modules/gear_presets/uscm.dm [hunk 4]
```diff
@@ -203,35 +166,11 @@
...
-		back_item = /obj/item/storage/backpack/marine/standard
-
-	new_human.equip_to_slot_or_del(new back_item(new_human), WEAR_BACK)
-
 /datum/equipment_preset/uscm/sg/forecon/lesser_rank
 	paygrade = "ME4"
 
 //*****************************************************************************************************/
 
-/datum/equipment_preset/uscm/sg/full
```
Comment (AndroBetel) [inline code/modules/gear_presets/uscm.dm:218]: why is this being removed....

**Review by AndroBetel — CHANGES_REQUESTED** [review]
i got ahead of myself actually...

File: code/modules/gear_presets/uscm.dm (hunk 4 above)
Comment (DexterDude) [reply code/modules/gear_presets/uscm.dm:218]: it doesnt do anything for the most part, all it did was make it so when you spawned a naked sg it wojld sometimes spawn an equipped marine anyway

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: code/modules/gear_presets/uscm_ship.dm (hunk 3 above)
Comment (DexterDude) [reply code/modules/gear_presets/uscm_ship.dm:593]: fail

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by AndroBetel — APPROVED** [review]

**Pull request merged**